### PR TITLE
Report page render times

### PR DESCRIPTION
**Pull request by frankharkins**
I'm using this for testing but figured it might be helpful to report in our weekly checks too.

### CODE REVIEW

File: scripts/commands/checkPagesRender.ts [hunk 1]
```diff
@@ -83,17 +83,27 @@ zxMain(async () => {
...
+    } else {
       console.error(`❌ Failed to render: ${fp}`);
       failures.push(fp);
     }
 
     // This script can be slow, so log progress every 10 files.
     if (numFilesChecked % 10 == 0) {
-      console.log(`Checked ${numFilesChecked} / ${files.length} pages`);
+      const averageTime = (
+        renderTimes.reduce((a, b) => a + b, 0) / renderTimes.length
```
Comment (Eric-Arellano) [inline scripts/commands/checkPagesRender.ts:101]: Lodash has a `mean()` function https://www.geeksforgeeks.org/lodash-_-mean-method/

File: scripts/commands/checkPagesRender.ts [hunk 2]
```diff
@@ -83,17 +83,27 @@ zxMain(async () => {
...
+      ).toFixed(0);
+      console.log(
+        `Checked ${numFilesChecked} / ${files.length} pages (~${averageTime}ms per page)`,
```
Comment (Eric-Arellano) [inline scripts/commands/checkPagesRender.ts:104]: Thoughts on instead computing the average for the batch of 10 files? That could help us to understand if certain files are acting up, like if we have outliers. This was indeed the problem with the static site building.

**Review by Eric-Arellano — COMMENTED** [review]
Great idea!

File: scripts/commands/checkPagesRender.ts (hunk 2 above)
Comment (frankharkins) [reply scripts/commands/checkPagesRender.ts:104]: That's what I wanted it to do, I reset `renderTimes` in the L106

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**